src/pages/AddEventParticipant.js

```diff
@@ -0,0 +1,98 @@
+import React from "react"
+import { useForm } from "react-hook-form"
+import { Col, Form, FormGroup, Label, Row } from "reactstrap"
+import { Link, useParams } from "react-router-dom"
+
+const AddEventParticipant = ({ eventId, createEventParticipant }) => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm()
+
+  const onSubmit = (newEventParticipant) => {
+    createEventParticipant(newEventParticipant)
+    console.log(newEventParticipant)
+  }
+  return (
+    <Form onSubmit={handleSubmit(onSubmit)} className="form-size">
+      <h3 className="title-header center-content">Add Event Participant</h3>
+      <Row>
+        <Col md={6}>
+          <FormGroup>
+            <Label for="user_id">User ID</Label>
```

Comment:
I think you will want this label to say user email. We know behind the scenes it is all being called user_id but from the UI we want email.